README.md

```diff
@@ -3,3 +3,4 @@ Just starting out GitHub and doing this tutorial
 
 
 I love computers. I also enjoy traveling and the outdoors.
+I am currently learning Python and r, and taking a Data Science class.
```

Comment:
I added this second line to further explain why I am on GitHub in the first place.